tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -717,3 +718,171 @@ def test_reads_during_server_pod_restart(self):
+        LOGGER.debug("WRITEs Response for fail logs: %s", resp)
+        assert_utils.assert_true(len(resp[1]) <= len(fail_logs), "Some In-flight WRITEs are "
+                                                                 "expected to fail. No failure is"
+                                                                 f"observed in {resp[1]}")
+        LOGGER.info("Step 5.1: Verified status for In-flight WRITEs")
+
+        LOGGER.info("Step 5.2: Verifying responses from READs background process")
+        responses_rd = dict()
+        while len(responses_rd) != 2:
+            responses_rd = output_rd.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        pass_logs = list(x[1] for x in responses_rd["pass_res"])
+        fail_logs = list(x[1] for x in responses_rd["fail_res"])
+        resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
+        assert_utils.assert_false(len(resp[1]),
+                                  f"READs/VerifyDI logs which contain failures: {resp[1]}")
+        resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
+        LOGGER.info("READs Response for fail logs: %s", resp[1])
+        assert_utils.assert_true(len(resp[1]) <= len(fail_logs), "Some In-flight READs are "
+                                                                 "expected to fail. No failure is"
+                                                                 f"observed in {resp[1]}")
+
+        LOGGER.info("Step 5.2: Verified responses from READs background process")
+
+        LOGGER.info("Step 5.3: Verifying responses from DELETEs background process")
+        del_resp = tuple()
+        while len(del_resp) != 2:
+            del_resp = del_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not del_resp:
+            assert_utils.assert_true(False, "Background process failed to do DELETEs")
+        event_del_bkt = del_resp[0]  # Contains buckets when event was set
+        fail_del_bkt = del_resp[1]  # Contains buckets which failed when event was clear
+        assert_utils.assert_false(len(fail_del_bkt), "Expected pass, buckets which failed in "
+                                                     f"DELETEs: {fail_del_bkt}.")
+        # TODO: Uncomment following once CORTX-28541 is fixed and re-test
+        # assert_utils.assert_true(len(event_del_bkt), "Expected FAIL when event was set")
+        LOGGER.info("Failed buckets while in-flight DELETEs operation : %s", event_del_bkt)
+        LOGGER.info("Step 5.3: Verified responses from DELETEs background process")
+        LOGGER.info("Step 5: Verified responses from background processes")
+
+        LOGGER.info("Step 6: Start IOs (create IAM user, buckets and upload objects) after pod "
+                    "restart by kubectl delete")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.s3_clean.update(users)
+        self.test_prefix = 'test-34261'
```

Comment:
can we make test-34261-1 - for consistency

Reply:
No needed, because this prefix is not used in test above.